src/content/en/updates/2019/06/layoutNG.md

```diff
@@ -0,0 +1,328 @@
+description: A new layout engine for Chromium
+
+{# wf_published_on: 2019-06-06 #}
+{# wf_updated_on: 2019-06-06 #}
+{# wf_featured_image: /web/updates/images/2019/06/layoutNG-header.jpg #}
+{# wf_tags: new-in-chrome,layout,performance #}
+{# wf_featured_snippet: LayoutNG is a new layout engine for Chromium that has been designed for the needs of modern scalable web applications. It improves performance isolation, better supports scripts other than Latin, and fixes many float, margin, and web compatibility issues. #}
+{# wf_blink_components: N/A #}
+
+# LayoutNG {: .page-title }
+
+{% include "web/_shared/contributors/emilaeklund.html" %}
+
+<style>
+  .w-figures-2-up {
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+    margin: 3rem 0;
+  }
+
+  .w-figures-2-up figure {
+    text-align: center;
+  }
+</style>
+
+Scheduled to be released in Chrome 76, LayoutNG is a new layout engine exiting a 
+multi-year effort. There are several exciting immediate improvements, 
+and additional performance gains and advanced layout features will be coming.
+
+## What's New?
+
+1. Improves performance isolation.
+1. **Better support** for scripts other than Latin
+1. **Fixes** many issues around floats and margins
+1. **Fixes** a large number of web compatibility issues
+
+Please note that LayoutNG will be **launched in stages**. In Chrome 76, LayoutNG 
+is used for inline and block layout. Other layout primitives 
+(such as table, flexbox, grid, and block fragmentation) will be replaced in subsequent releases.
+
+### Developer visible changes
+Although the user visible impact should be minimal, LayoutNG changes some behavior 
+in very subtle ways, fixes hundreds of tests, and improves c
+ompatibility with other browsers. Despite our best efforts, it is likely that this 
+will cause some sites and applications to render or behave slightly differently.
+
+The performance characteristics are also quite different; although performance on 
+a whole is similar or slightly better than before, certain use 
+cases are likely to see performance improvements, while others are expected to 
+regress somewhat, at least short-term.
+
+## Floats
+LayoutNG reimplements support for floating elements (`float: left;` and `float: right;`) 
+fixing a number of correctness issues around placement 
+of floats in relation to other content.
+
+#### Superimposed Content
+The legacy float implementation didn’t correctly account for margins when placing 
+content around a floating element, resulting in the content 
+partially or fully overlapping the float itself. The most common manifestation of 
+this bug appears when an image is positioned beside a paragraph 
+where the avoidance logic fails to account for the height of a line. 
+(See [Chromium bug #861540](https://crbug.com/861540).)
+
+<div class="w-figures-2-up">
```

Comment:
To do two side by side figures, you should:
```
<figure class="attempt-left">
...
</figure>
<figure class="attempt-right">
...
</figure>
<div class="clearfix"></div>
```

The nice thing about that method is that it works responsively, on wide screens it'll be side by side and it'll then go inline on smaller screens.

Reply:
Please fix throughout